fhir-examples/src/main/resources/json/profiles/fhir-ig-davinci-pdex/Bundle-1000000-1.json

```diff
@@ -0,0 +1 @@
"http://terminology.hl7.org/3.0.0/CodeSystem-provenance-participant-type.html\">Provenance participant type</a>#author)</span></td><td><a href=\"Organization-ProviderOrg1.html\">Organization/ProviderOrg1</a> \"Provider 1\"</td></tr></table><h3>Entities</h3><table class=\"grid\"><tr><td>-</td><td><b>Extension</b></td><td><b>Role</b></td><td><b>What</b></td></tr><tr><td>*</td><td></td><td>source</td><td><span>: No reference available</span></td></tr></table></div>"},"target":[{"reference":"Encounter/7-1"}],"recorded":"2020-07-10T16:26:23.217+00:00","agent":[{"type":{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/provenance-participant-type","code":"author","display":"Author"}]},"who":{"reference":"Organization/ProviderOrg1"}}],"entity":[{"extension":[{"url":"http://hl7.org/fhir/us/davinci-pdex/StructureDefinition/ProvenanceSourceFrom","valueCodeableConcept":{"coding":[{"system":"http://hl7.org/fhir/us/davinci-pdex/CodeSystem/ProvenancePayerDataSource","code":"hl7ccda","display":"HL7 C-CDA"}]}}],"role":"source","what":{"display":"No reference available"}}]},"response":{"status":"200"}},{"fullUrl":"https://example.com/base/Provenance/1000004-1","resource":{"resourceType":"Provenance","id":"1000004-1","meta":{"versionId":"1","lastUpdated":"2019-12-31T20:26:23.217+00:00"},"text":{"status":"generated","div":"<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Generated Narrative</b></p><div style=\"display: inline-block; background-color: #d9e0e7; padding: 6px; margin: 4px; border: 1px solid #8da1b4; border-radius: 5px; line-height: 60%\"><p style=\"margin-bottom: 0px\">Resource \"1000004-1\" Version \"1\" Updated \"2019-12-31 08:26:23+0000\" </p></div><p><b>target</b>: <a href=\"Bundle-1000000-1.html\">Bundle/1000000-1</a></p><p><b>recorded</b>: Dec 31, 2019, 8:26:23 PM</p><h3>Agents</h3><table class=\"grid\"><tr><td>-</td><td><b>Type</b></td><td><b>Role</b></td><td><b>Who</b></td></tr><tr><td>*</td><td>Transmitter <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"http://hl7.org/fhir/us/core/STU3.1/CodeSystem-us-core-provenance-participant-type.html\">US Core Provenance Participant Type Extension Codes</a>#transmitter)</span></td><td>Transcriber <span style=\"background: LightGoldenRodYellow; margin: 4px; border: 1px solid khaki\"> (<a href=\"http://terminology.hl7.org/3.0.0/CodeSystem-contractsignertypecodes.html\">Contract Signer Type Codes</a>#TRANS)</span></td><td><a href=\"Organization-Payer1.html\">Organization/Payer1</a> \"Payer 1\"</td></tr></table></div>"},"target":[{"reference":"Bundle/1000000-1"}],"recorded":"2019-12-31T20:26:23.217+00:00","agent":[{"type":{"coding":[{"system":"http://hl7.org/fhir/us/core/CodeSystem/us-core-provenance-participant-type","code":"transmitter","display":"Transmitter"}]},"role":[{"coding":[{"system":"http://terminology.hl7.org/CodeSystem/contractsignertypecodes","code":"TRANS"}]}],"who":{"reference":"Organization/Payer1"}}]},"response":{"status":"200"}}]}
```

Comment:
we've started to move spec examples out of fhir-examples and into the `src/test/resources` of the `fhir-ig-` project that wraps the actual IG

Reply:
Done in https://github.com/IBM/FHIR/pull/3424.